packages/deploy-tooling/README.md

```diff
@@ -101,6 +101,8 @@ Options:
   --username                           ABAP Service username
   --password                           ABAP Service password
   --transport <transport-request>      Transport number to record the change in the ABAP system
+  --create-transport                   Create a transport request during deployment
+  --package <abap-package>             Package name for deploy target ABAP system (only required when --create-transport is used)
```

Comment:
The `package` shouldn't be required for undeploy. The deployed app is already part of a package and undeployment doesn't need to know which on (I think).

Reply:
The `package` is a required param for `createTransportRequest` 
https://github.com/SAP/open-ux-tools/blob/84579be8d6780b5133524cc66b3aab1dc68a2d83/packages/axios-extension/src/abap/adt-catalog/services/transportrequest-service.ts#L7-L12

Reply:
@slin-sap you are the ADT expert here, is it an issue with the type or is a package name really required for creating a transport to undeploy an existing BSP?

Reply:
Weird but ok, retracting my comment then.